src/styles/utils/variables.scss

```
@@ -0,0 +1,27 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap';
+
+$accent-color: #00acdc;
+$dark-text-color: #060b35;
+$light-text-color: #616070;
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+}
+
+h2 {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
```

These are not variables, so move these styles to the separate file
```suggestion
*,
*::before,
*::after {
  box-sizing: border-box;
}

body {
  margin: 0;
  font-family: Roboto, sans-serif;
  font-size: 12px;
  font-weight: 500;
  line-height: 14px;
}

h2 {
  margin: 0;
}

p {
  margin: 0;
}
```